examples/assets/scripts/animation/tween.js

```diff
@@ -0,0 +1,318 @@
+var Tween = pc.createScript('tween');
+
+Tween.attributes.add('tweens', {
+    type: 'json',
+    schema: [
+        {
+            name: 'autoPlay',
+            title: 'Autoplay',
+            description: 'Play tween immediately.',
+            type: 'boolean',
+            default: false
```

Comment:
Hey @vkalpias - I can't get defaults working for array-based JSON attributes. Can you provide an example? I'd like to update the User Manual to cover this topic as well.

Reply:
I believe the default is set on the JSON attribute, not the individual schema fields. e.g.:

```
Test.attributes.add('json',{
    type: 'json',
    schema: [{
        name: 'field1',
        type: 'number'
    }, {
        name: 'field2',
        type: 'string'
    }],
    default: {
        field1: 4,
        field2: 'hello world'
    }
});

```

Reply:
So this is wrong? https://developer.playcanvas.com/en/user-manual/scripting/script-attributes/#json-attribute

Reply:
And that also works for `array: true`?

Reply:
@vkalpias The defaults on the schema work fine in the Editor:
```
var Sandbox = pc.createScript('sandbox');
Sandbox.attributes.add('settings', {
    type: 'json',
    schema: [{
        name: 'gravity',
        type: 'number',
        default: -9.8
    }, {
        name: 'startingHealth',
        type: 'number',
        default: 20
    }, {
        name: 'godMode',
        type: 'boolean',
        default: false
    }]
});
```

Reply:
Yep, it just doesn't work for the engine-only case.

Reply:
Defaults now fixed here: https://github.com/playcanvas/engine/pull/2525  Resolving...